packages/app-content-pages/src/shared/components/AboutHeader/AboutHeader.js

```diff
@@ -52,28 +51,33 @@ function AboutHeader () {
   ]
 
   return (
-    <Box align='center' as='header' background='brand'>
```

Comment:
Where is the page header now? I couldn't find it in the code changes for this PR.

Reply:
For comparison, here's a screenshot of the DOM for a project Research page, which has a `<header>`, `<aside>` and `<main>`.

<img width="800" alt="Screenshot of the DOM inspector for the Planet Hunters - TESS Research page, showing the <header>, <aside> and <main> landmark elements." src="https://github.com/zooniverse/front-end-monorepo/assets/59547/34545377-a26a-40cf-bc55-99ba84ddf098">

Reply:
Good catch! I added a header element to AboutHeader

Reply:
👍 

The Zooniverse header should be inside the page `<header>` too, but I think we could have two headers if that makes the layout easier. If we have two headers, we should probably give them labels to tell them apart. Projects wrap both the Zooniverse header and project header in a single `<header>`.

The page's main content should be wrapped in `<main>` too, ideally with the page's `h1` heading as its heading. I don't think that's needed for this PR, though.

Reply:
Oohhh you're right project pages put both components inside the `<header>`. With our plan to put ZooHeader inside app-root, maybe we should consider also handling the ProjectHeader or AboutHeader in app-root too... 🤔 Or would that start to defeat the purpose of separating out library packages from app-root hmmm

Reply:
I know pages can only have one `<main>` landmark, but I think multiple headers are allowed, so that you can have site-wide headers but also section headers.

It might be worth looking at how bbc.co.uk (or a similar big site) structures their pages.

Reply:
I do know that `ZooHeader` isn't wrapped in `<header>` itself, so that consuming apps have the flexibility to decide how to lay out page headers.

Reply:
BBc News has two headers. The top one is labelled 'BBC-wide' (highlighted here.) The red header underneath it is labelled 'News'. So we could do this and have more flexibility in our layouts.

<img width="800" alt="The DOM inspector showing the headers for the BBC News landing page. The BBC-wide header is hightlighted." src="https://github.com/zooniverse/front-end-monorepo/assets/59547/0925272a-2a9b-47d6-9d34-99d9b3040309">

Reply:
I should add a labelled site header to #5459 too.

Reply:
Good to know thanks!